pydis_site/apps/content/resources/guides/pydis-guides/contributing/bot.md

```diff
@@ -37,21 +36,115 @@ If your bot fails to start with a `PrivilegedIntentsRequired` exception, this in
 
 ### Configure the Bot
 You now have both the bot's code and a server to run it on. It's time you to connect the two by changing the bot's configurations.
+This can be done either automatically or manually, and we'll be detailing the steps for both.
+
+#### Automatic configuration
+To make setup much easier, there is a file called `botstrap.py` that represents a script to bootstrap the configuration for you and help you get started immediately
+without having to spend much time copying ids from your newly created server into your configuration file.
+
+**Note**: This phase can be skipped and done manually, but would require extra manual work.
+
+##### 1. Script setup
+##### 1.1. Environment variables
+You will need to create a file called `.env` which will contain two required values for the script to work: `BOT_TOKEN` and `GUILD_ID`
+
+Inside, add the following two lines:
+
+```text
+BOT_TOKEN=YourDiscordBotTokenHere
+GUILD_ID=YourDiscordTestServerIdHere
+```
+See [here](../creating-bot-account) for help with obtaining the bot token and [here](../obtaining-discord-ids#guild-id) for help with obtaining the guild's id
+
+**Note**: The `.env` is and should remain ignored by git, otherwise you risk pushing sensitive information.
+##### 1.2 Setting up the script environment
+The bootstrapping script is a Python program so you will need a compatible Python version and the necessary dependencies installed,
+which are all detailed here:
+
+1. Make sure you follow steps `1` and `2` [here](#setting-up-a-development-environment)
+2. [Install the `config-bootstrap` dependency group](../installing-project-dependencies#installing-specific-dependency-groups).
+
+#### 2. Running the script
+
+Once the script setup phase is complete, all that is left is to run it.
+To do this, you'll simply need to run the `configure` poetry task like this
+
+```shell
+$ poetry run task configure
+```
+
+or, without poetry and from the root directory
+
+```shell
+python3 -m botstrap
+```
+
+Once the script has finished running, you'll notice the creation of a new file called [`.env.server`](#envserver) at your project's root directory.
```

Comment:
I think `at` indicates that it will be exactly in that position, whereas in will infer that it is contained within the root directory ( and its subfolders) and not necessarily at the top level position